spec/requests/users_controller_spec.rb

```diff
@@ -3674,11 +3679,9 @@ def create_and_like_post(likee, liker)
       end
 
       describe "include_post_count_for" do
-
-        fab!(:admin) { Fabricate(:admin) }
         fab!(:topic) { Fabricate(:topic) }
 
-        before do
+        before_all do
```

Comment:
I don't know if this one is worth it. When I see `before_all` I think: potential test leak. 😅

Reply:
It's still in a transaction. It's really no different than `fab!`, except the posts aren't named. N.B. `before_all` is different than `before(:all)`. https://github.com/test-prof/test-prof/blob/master/docs/recipes/before_all.md